### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by someaddons**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
-Fix fisher interaction text for not finding water
-Fix citizen glowing not working

[ x] Yes I tested this before submitting it.
[ ] I also did a multiplayer test.

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/manual_en_us.json [hunk 1]
```diff
@@ -1376,7 +1376,7 @@
   "com.minecolonies.gui.visiblestatus.digging": "Digging a grave",
   "com.minecolonies.gui.visiblestatus.burying": "Burying a citizen",
 
-  "entity.fisherman.messagewatertoofar": "My hut seems to be too far away from water, and I can't find any water to fish in! It should be an area of at least 7x7 blocks, and the same height as my hut.",
+  "entity.fisherman.messagewatertoofar": "I can't find any water to fish in! There should be an area of at least 7x7x2 water blocks nearby.",
```
Comment (Raycoms) [inline src/main/resources/assets/minecolonies/lang/manual_en_us.json]: maybe something more verbose, like "7 wide, and 7 long and at least 2 deep" or so

Comment (Talyda) [reply src/main/resources/assets/minecolonies/lang/manual_en_us.json]: Does it no longer need to be the same Y level as the hutblock?

Comment (Raycoms) [reply src/main/resources/assets/minecolonies/lang/manual_en_us.json]: no

Comment (Talyda) [reply src/main/resources/assets/minecolonies/lang/manual_en_us.json]: Awesome! Thanks! My fisher design can happen! =D

Comment (uecasm) [reply src/main/resources/assets/minecolonies/lang/manual_en_us.json]: Being at the same y-level wasn't a requirement even in 1.19.  idk when it was actually fixed, but it was long long ago.

**Comment by Raycoms** [comment]
Yeah, it was mostly the pathfinding limit having a hard time tbh, it was never a hard limit

-------- Original Message --------

**Pull request merged**